bin/_mkproject.bat

```diff
@@ -24,8 +24,8 @@ if "%2"=="" goto :missing_name
 if "%3"=="" goto :missing_silo
 
 set PROJECTDIR=%1%2
-set curPROJECT=%1
-set curPROJECTNAME=%2
+set CURRENTPROJECT=%1
```

Comment:
Excellent! Would it be possible to add a comment about what these are for? For our future selves, and other developers coming along.